packages/arb-ts/src/lib/l2Bridge.ts

```diff
@@ -118,75 +139,34 @@ export class L2Bridge {
     ](erc20l1Address, to, amount, '0x', overrides)
   }
 
-  public async updateAllL2Tokens(): Promise<Tokens> {
-    for (const l1Address in this.l2Tokens) {
-      const l2Address = this.l2Tokens[l1Address]?.ERC20?.contract.address
-      if (l2Address) {
-        await this.getAndUpdateL2TokenData(l1Address, l2Address)
-      }
-    }
-    return this.l2Tokens
-  }
-
-  public async getAndUpdateL2TokenData(
+  public async getL2TokenData(
     erc20L1Address: string,
```

Comment:
this param is currently usused

Reply:
It might (indeed) make more sense to use the L2 address as the (only) input for this method (but currently elsewhere, i.e., the tests, it's still assumed to be the L1 address)